docs/kerberos.md

```diff
@@ -146,13 +151,25 @@ You can also set the base64 encoded krb5.conf after install time:
 **Note** This setting `SPARK_MESOS_KRB5_CONF_BASE64` will overwrite/override any settings set with
 `SPARK_SECURITY_KERBEROS_KDC_HOSTNAME`, `SPARK_SECURITY_KERBEROS_KDC_PORT`, and `SPARK_SECURITY_KERBEROS_REALM`
 
+### Setting the Spark User
+
+By default, when Kerberos is enabled, Spark runs as the OS user coresponding to the
+Kerberos principal. For example, the principal "alice@LOCAL" would likely translate to "alice".
```

Comment:
Suggestion: "For example, the principal "alice@LOCAL" would ~likely~ map to the username "alice"."
(The only edit I feel strongly about here is dropping "likely").